autoware_launch/launch/planning_simulator.launch.xml

```diff
@@ -47,6 +51,10 @@
       <!-- Map -->
       <arg name="lanelet2_map_file" value="$(var lanelet2_map_file)"/>
       <arg name="pointcloud_map_file" value="$(var pointcloud_map_file)"/>
+      <!-- Control -->
+      <arg name="enable_obstacle_collision_checker" value="$(var enable_obstacle_collision_checker)"/>
+      <arg name="lateral_controller_mode" value="$(var lateral_controller_mode)"/>
+      <arg name="longitudinal_controller_mode" value="$(var longitudinal_controller_mode)"/>
```

Comment:
Could you check whether the option works when launching planning_simulator.launch.xml even if we discard the change of planning_simulator.launch.xml.

Reply:
"discard the change of planning_simulator.launch.xml": you mean that different options in autoware.launch.xml and planning_simulator.launch.xml right? Do we want to test that planning_simulator.launch.xml can overwrites the autoware.launch.xml properly?

Reply:
No the different options.

This is an example of tier4/autoware_launch
[cruise_planner_type in autoware.launch.xml](https://github.com/tier4/autoware_launch/blob/e282e15d4bddcebec6401ecdc5cf525a71cfff20/autoware_launch/launch/autoware.launch.xml#L14) can be designated when launching [planning_simualtor.launch.xml](https://github.com/tier4/autoware_launch/blob/awf-latest/autoware_launch/launch/planning_simulator.launch.xml) like `ros2 launch autoware_launch planning_simulator.launch.xml cruise_planner_type:=obstacle_stop_planner` even though planning_simulator.launch.xml does not have an argument of `cruise_planner_type`.

Therefore, in the same way, in this PR, we do not have to add longitudinal_controller_mode in planning_simulator.launch.xml.

Reply:
Oh I see, I tested both scenarios (with removing `lateral_controller_mode` from planning_simulator.launch.xml and without removing it). We can assign mode from terminal (like you said), and it worked.

Reply:
So could you remove `lateral_controller_mode` from launch files other than `autoware.launch.xml`. It worked well, right?

Reply:
Yes, I removed from planning_simulator.launch.xml and worked.

Reply:
Sorry for misunderstanding, my college just warned me. I removed options from  `planning_simulator.launch.xml` and pushed new commit. Would you check it again?